MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,345 @@
+# Migration from Splunk Connect for Kubernetes
+
+## Assumptions
+
+You are running Splunk Connect for Kubernetes (SCK) 1.4.9 and want to migrate to Splunk OpenTelemetry Collector for Kubernetes. Previous versions of the SCK are out of scope, but this guide still applies. You might encounter some version specific issues (such as missing/changed config options) but that shouldn't prevent you from proceeding with migration.
+
+## Changes in components
+
+SCK has 3 components/applications:
+
+1. Logs, metrics and traces from a Kubernetes cluster (deployed as a DaemonSet)
+2. Application to fetch cluster metrics from a kubernetes cluster (deployed as a deployment)
+3. Application to fetch kubernetes objects metadata from a kubernetes cluster (deployed as a deployment)
+
+All SCK applications use Fluentd to work with logs, metrics and objects. Fluentd has significant performance issues when used to fetch logs in a Kubernetes cluster with pods that have very high throughput.
+
+Splunk OpenTelemetry Collector for Kubernetes provides significant performance improvements over the SCK through use of the OpenTelemetry Collector agent and native OpenTelemetry functionality for logs collection rather than Fluentd. You can learn more about the performance characteristics of this new application [Performance of native OpenTelemetry logs collection](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#performance-of-native-opentelemetry-logs-collection).
+
+Splunk OpenTelemetry Collector for Kubernetes has the following components and applications:
+
+1. Splunk OpenTelemetry Collector Agent (`agent`) to fetch logs, metrics and traces from a kubernetes cluster (deployed as a Kubernetes DaemonSet)
+2. Splunk OpenTelemetry Collector Cluster Receiver (`clusterReceiver`) to fetch metrics from a Kubernetes API (deployed as a Kubernetes 1-replica Deployment).
+3. Optional Splunk OpenTelemetry Collector Gateway (`gateway`) to forward data through it to reduce load on Kubernetes API and apply additional processing (deployed as a Kubernetes Deployment).
+
+There is no application available for fetching Kubernetes objects metadata from a Kubernetes cluster.
+
+### Changes in logs, metrics, and objects
```

Comment:
logs, metrics, and objects

Reply:
what do you mean here?

Reply:
nvm, deleting this whole table as suggested